.gitmodules

```diff
@@ -4,8 +4,8 @@
 	branch = main
 [submodule "ccpp-physics"]
 	path = ccpp/physics
-	url = https://github.com/NCAR/ccpp-physics
-	branch = main
+	url = https://github.com/grantfirl/ccpp-physics
+	branch = ufs-dev-PR216
```

Comment:
Just double checking that pointing to your `ufs-dev-PR216` branch is on purpose. (I'm guessing until the CCPP Physics [PR#1092](https://github.com/NCAR/ccpp-physics/pull/1092) is merged?) Everything else looks good to me, thanks!

Reply:
Yes, it's just for testing, just like for UFS. It gets reverted after the ccpp-physics PR is merged.